src/migrations/1598829375940-TypeORMMetadata.ts

```diff
@@ -0,0 +1,14 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
This was autogenerated too right????

Reply:
the structure was generated through migration:create but I handwrote the sql query